soundfile.py

```diff
@@ -1096,8 +1273,47 @@ def buffer_write(self, data, dtype):
         assert written == frames
         self._update_frames(written)
 
-    def blocks(self, blocksize=None, overlap=0, frames=-1, dtype='float64',
-               always_2d=False, fill_value=None, out=None):
+
```

Comment:
see above

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/29c215fb43d8b7640f8634f68586ef07dbb79294